src/main.cpp

```diff
@@ -1,21 +1,30 @@
 #include <M5StickC.h>
-#include <WiFi.h>
+#include <ArduinoJson.h>
 #include "Wi-Fi.h"
+#include "communication.hpp"
+#include "key.cpp"
 
-const char *ssid = wifi_ssid;
-const char *password = wifi_password;
-
-int PIN = 36;
+char *ssid = wifi_ssid;
+char *password = wifi_password;
+char *url = host_url;
+Communication com;
```

Comment:
```suggestion
PostAPI api;
```

Communicationと言われてAPIと通信するように思えないのでPostAPIとかにしてください